activities/membership-growth/membership-proposal-template.md

```diff
@@ -0,0 +1,116 @@
+
+The Foundation for Public Code is currently working with over 20 public organizations around Europe (including small municipalities, big cities, regional and national governments) on an open and collaborative process for developing reusable software components. Its aim is to help public organizations procure, develop and maintain code together.
+
+For this a [Standard for Public Code](https://standard.publiccode.net/) has been developed. This is a collection of principles to help policy makers, managers, designers, developers and vendors work together to make code that is easy to understand, reuse and maintain.
+
+It also offers [codebase stewardship activities](https://about.publiccode.net/activities/codebase-stewardship/activities.html), which include a) reviewing the codebase (and all new additions to it) to make sure it complies with the Standard for Public Code, b) facilitating multi-stakeholder development between public and private organizations, c) helping with marketing to ensure public organizations understand the software and how to implement it, and d) supporting private sector parties to work effectively on these public-sector led software systems.
+
+This will:
+
+* enable higher quality public services
+* enable the pooling of resources with other public bodies to share the cost of development, maintenance and upgrading
+* build in agility for future growth paths and innovation
+* make solutions more reusable
+* gain international exposure
+
+## What becoming a member means
+
+Becoming a member means steering and becoming a co-owner of the association. This includes shaping the governance, operations and financial structure of the Foundation for Public Code.
+
+Members receive voting rights at the General Assembly, where voting rights are proportional to your member dues. For example, members vote on which codebases are stewarded by the Foundation for Public Code.
+
+Membership also includes onboarding support to help you get the most of your membership. During the onboarding we help you with your public code projects.
+
+Onboarding includes:
+
+* supporting your public code projects and codebases
+* introducing you to our network and codebase communities
+* preparing you for your first general assembly
+
+And supporting you with your codebases on:
+
+* quality of your codebases - understanding how to make them more modular and reusable
+* community for your codebases - understanding how to collaborate better and more openly with other organizations
+* product packaging of your codebases - understanding how to better market and promote your codebases
+* support of your codebases - understanding how to improve your collaboration with the market
+* [assessment of your codebases](https://about.publiccode.net/activities/codebase-stewardship/criteria-for-codebase-stewardship.html) for codebase stewardship
+
+As an association, our resources are allocated across members. Our support for each member is subject to availability.
```

Comment:
"subject to availability" was added when we offered one-off pre-membership onboarding support. It might be more accurate now to say:

```suggestion
As an association, our members determine how our resources are allocated across members and potential members.
```